frontend/src/components/Typography/Caption/index.module.css

```diff
@@ -0,0 +1,7 @@
+.caption {
+  font-family: var(--font-family-body);
+  font-size: var(--font-size-xxs);
+  font-weight: 400;
+  line-height: 16px;
+  color: var(--color-blue);    
+}
```

Comment:
Missing blank line before end of line. Are you running the linters?

Reply:
Will fix! Got used to only run style because vsc warns be about prettier